mapbook-iOS/AppDelegate.swift

```diff
@@ -110,3 +103,84 @@ extension UIColor {
     }
 }
 
+extension Bundle {
```

Comment:
Should these extensions be granted their own homes? E.g. `Bundle+ArcGISRuntime.swift` and `Bundle+AppInfo.swift` etc.?

Reply:
I'll break out these extensions.